bloom/generators/debian/generator.py

```diff
@@ -575,7 +575,7 @@ def sanitize_package_name(name):
 class DebianGenerator(BloomGenerator):
     title = 'debian'
     description = "Generates debians from the catkin meta data"
-    has_run_rosdep = False
+    has_run_rosdep = os.environ.get('BLOOM_SKIP_ROSDEP_UPDATE', '0') not in ['0', 'f', 'false', 'n', 'no']
```

Comment:
I'd recommend:

```suggestion
    has_run_rosdep = os.environ.get('BLOOM_SKIP_ROSDEP_UPDATE', '0').lower() not in ['0', 'f', 'false', 'n', 'no']
```

But it is certainly optional.